File: test/pizzapazza/ui/panel/TestFillingPanel.java
package pizzapazza.ui.panel;

import javascript.awt.BorderLayout;
import javascript.swing.JSFrame;
import javascript.swing.JSPanel;

/**
 *
 * @author gianpiero.diblasi
 */
public class TestFillingPanel extends JSFrame {
  
  public TestFillingPanel() {
    super();
    
    Z4FillingPanel fillingPanel = new Z4FillingPanel();
    fillingPanel.setSize(500, 300);
    
    JSPanel p = new JSPanel();
    p.add(fillingPanel, null);
    this.getContentPane().add(p, BorderLayout.NORTH);
  }
}

File: transpile-test/pizzapazza/ui/panel/TestFillingPanel.js
/**
 * @author gianpiero.diblasi
 */
class TestFillingPanel extends JSFrame {

  constructor() {
    super();
    let p = new JSPanel();
    p.add(new Z4FillingPanel(), null);
    this.getContentPane().add(p, BorderLayout.NORTH);
  }
}
